mlflow/pyfunc/__init__.py

```diff
@@ -628,6 +632,39 @@ def __repr__(self):
         return yaml.safe_dump({"mlflow.pyfunc.loaded_model": info}, default_flow_style=False)
 
 
+def _warn_dependency_requirement_mismatches(model_path):
+    """
+    print warning when installed model dependency package versions mismatch with the
+    model requirements file.
+    """
+    req_file_path = os.path.join(model_path, _REQUIREMENTS_FILE_NAME)
+    if not os.path.exists(req_file_path):
+        return
+
+    try:
+        mismatch_errs = []
```

Comment:
```suggestion
        mismatch_infos = []
```

`errs` sounds like this is an array of exceptions.